src/hotspot/share/opto/divnode.cpp

```diff
@@ -1518,7 +1518,8 @@ Node* ModFNode::Ideal(PhaseGVN* phase, bool can_reshape) {
   PhaseIterGVN* igvn = phase->is_IterGVN();
 
   bool result_is_unused = proj_out_or_null(TypeFunc::Parms) == nullptr;
-  if (result_is_unused) {
+  bool has_control_output = proj_out_or_null(TypeFunc::Control) != nullptr;
```

Comment:
Nit: Maybe replace this with `is_dead = proj_out_or_null(TypeFunc::Control) == nullptr;` and check for `!is_dead` below?

Reply:
Fine with me! At the very least, your name is more semantic, and less "here is a name that repeats what the code says".